lib/shopify_app/sessions_controller.rb

```diff
@@ -13,9 +13,14 @@ def create
     def callback
       if response = request.env['omniauth.auth']
         shop_name = response.uid
-        sess = ShopifyAPI::Session.new(shop_name, response['credentials']['token'])
+        token = response['credentials']['token']
+
+        sess = ShopifyAPI::Session.new(shop_name, token)
         session[:shopify] = ShopifyApp::SessionRepository.store(sess)
         session[:shopify_domain] = shop_name
+
+        WebhooksManager.run(shop_name, token) if ShopifyApp.configuration.has_webhooks?
```

Comment:
Without looking at the next file, I thought this was going to be doing inline HTTP calls. Maybe rename this to `run_later` or something along those lines?